db/migrate/20151013210331_change_phone_type_in_contacts.rb

```diff
@@ -0,0 +1,11 @@
+class ChangePhoneTypeInContacts < ActiveRecord::Migration
+  def change
+    def up
+      change_column :contacts, :phone, :string
+    end
+
+    def down 
```

Comment:
Method definitions must not be nested. Use `lambda` instead.<br>Trailing whitespace detected.